anonymizer/OpenDiffix.Service/CSVFormatter.fs

```diff
@@ -9,6 +9,9 @@ let private quoteString (string: string) =
 let private csvFormat value =
   match value with
   | Null -> ""
+  // Lowercase `Boolean` in order to be consistent with JavaScript rendering.
```

Comment:
I'm not happy about this, but the discrepancy (`true/false` in preview in Dfd vs `True/False` in exported CSV) was bugging me. It's slightly easier to fix this way around

Reply:
What does `JavaScript rendering` mean?

Reply:
hm, I guess what is happening when `true/false` becomes a string (`.toString()` I presume). I didn't find any code handling that in DfD, so I guess this is just the default. THat happens when we're rendering tables. I don't like the wording either, any suggestions?